### DISCUSSION TITLE
Не компилируется проект

### DISCUSSION THREAD

**Discussion by XDiopSideX** [vvip-68/GyverPanelWiFi#315] [Задавайте вопросы]
Добрый день, прошу помощи! 
Не могу скомпилировать проект, выдаёт ошибку. Всё делал по инструкции, библиотеки поставил из папки с проектом.

`In file included from c:\Users\Anton\Documents\Arduino\libraries\FastLED\src/FastLED.h:75,
                 from C:\Users\Anton\Downloads\GyverMatrixWiFi-master\GyverMatrixWiFi-master\firmware\GyverMatrixOS_v1.13\GyverMatrixOS_v1.13.ino:18:
c:\Users\Anton\Documents\Arduino\libraries\FastLED\src/fastspi.h:157:23: note: '#pragma message: No hardware SPI pins defined.  All SPI access will default to bitbanged output'
  157 | #      pragma message "No hardware SPI pins defined.  All SPI access will default to bitbanged output"
      |                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:\Users\Anton\Downloads\GyverMatrixWiFi-master\GyverMatrixWiFi-master\firmware\GyverMatrixOS_v1.13\clock.ino: In function 'long unsigned int sendNTPpacket(IPAddress&)':
C:\Users\Anton\Downloads\GyverMatrixWiFi-master\GyverMatrixWiFi-master\firmware\GyverMatrixOS_v1.13\clock.ino:60:1: error: no return statement in function returning non-void [-Werror=return-type]
   60 | }
      | ^
C:\Users\Anton\Downloads\GyverMatrixWiFi-master\GyverMatrixWiFi-master\firmware\GyverMatrixOS_v1.13\runningText.ino: In function 'uint8_t getFont(uint8_t, uint8_t)':
C:\Users\Anton\Downloads\GyverMatrixWiFi-master\GyverMatrixWiFi-master\firmware\GyverMatrixOS_v1.13\runningText.ino:100:1: error: control reaches end of non-void function [-Werror=return-type]
  100 | }
      | ^
cc1plus.exe: some warnings being treated as errors

exit status 1

Compilation error: no return statement in function returning non-void [-Werror=return-type]`

**Comment by vvip-68** [comment]
GyverMatrixOS ??? Я понятия не имею что это за проект. Вам лучше к автору обратиться. Это, наверное, Гайвер писал? К нему и стоит обращаться, я думаю...
